benchmark_runner/common/elasticsearch/elasticsearch_operations.py

```diff
@@ -22,20 +25,33 @@ class ElasticSearchOperations:
     MAX_SEARCH_RESULTS = 1000
     MIN_SEARCH_RESULTS = 100
 
-    def __init__(self, es_host: str, es_port: str, es_user: str = '', es_password: str = '', timeout: int = 2000):
+    def __init__(self, es_host: str, es_port: str, es_user: str = '', es_password: str = '', es_url_protocol: str = 'http', timeout: int = 2000):
```

Comment:
es_url_protocol: str = '' => Do you think it better ?
We dont want to expose our default method